modules/pytket-cirq/pytket/extensions/cirq/backends/cirq.py

```diff
@@ -54,9 +53,9 @@
 from pytket.backends.backendinfo import BackendInfo
 from pytket.backends.resulthandle import _ResultIdTuple
 from pytket.utils.results import KwargTypes
-from pytket.extensions.cirq import tk_to_cirq
+from pytket.extensions.cirq.cirq_convert import tk_to_cirq  # type: ignore
 from pytket.utils.outcomearray import OutcomeArray
-from .cirq_utils import _get_default_uids
+from .cirq_utils import _get_default_uids  # type: ignore
```

Comment:
This shouldn't need a type ignore. Do you know why it does? (especially since it seems it wasn't necessary before)

Reply:
you were right, think my local build is turbo cursed